Dockerfile

```diff
@@ -1,43 +1,36 @@
-FROM golang:1.14.5-alpine as builder
+FROM golang:1.13-alpine as builder
```

Comment:
Consider puting the `Dockerfile` changes in a separate commit.

Reply:
But also, why did you reduce the version of the golang alpine container?

Reply:
rebase mistake, will remove this!